ktor-swagger-root.gradle.kts

```diff
@@ -183,8 +184,8 @@ task("stage") {
     dependsOn(":ktor-sample-swagger:installDist")
 }
 
-task<Wrapper>("wrapper") {
-    gradleVersion = "4.8"
+tasks.withType<Wrapper>().configureEach {
```

Comment:
Why configure the wrapper this way instead of using the new `wrapper { ... }` syntax?

Reply:
Lazily evaluates it only if the `Wrapper` task is needed this way.